app/components/question-box.jsx

```diff
@@ -0,0 +1,85 @@
+// https://github.com/EnCiv/civil-pursuit/issues/100
+
+import React from 'react'
+import { createUseStyles } from 'react-jss'
+import StatusBadge from './status-badge'
+import Markdown from 'markdown-to-jsx'
+
+const QuestionBox = props => {
+  const { subject = '', description = '', participants = 0 } = props
```

Comment:
please add className and ...otherProps here. and then add them to the outer div.  See the react component guidelines - https://github.com/EnCiv/civil-pursuit?tab=readme-ov-file#react-component-guidelines-and-notes and click the > to expand the summary.